src/main/java/vazkii/botania/common/crafting/RecipeBrew.java

```diff
@@ -143,7 +144,7 @@ public boolean equals(Object o) {
 		public RecipeBrew read(@Nonnull Identifier id, @Nonnull JsonObject json) {
 			String brewStr = JsonHelper.getString(json, "brew");
 			Identifier brewId = Identifier.tryParse(brewStr);
-			if (brewId == null || !BotaniaAPI.instance().getBrewRegistry().containsId(brewId)) {
+			if (brewId == null || BotaniaAPI.instance().getBrewRegistry().get(brewId) == ModBrews.fallbackBrew) {
```

Comment:
can you explain this and the one below? The existing check should work.

Reply:
```java
@Environment(EnvType.CLIENT)
public abstract boolean containsId(Identifier id);
```

Reply:
I could replace these with `Registry.BLOCK.get(Registry.BLOCK.getDefaultId());` if you're unsure about the hardcoding

Reply:
Hmm, is there a method that returns null instead of the default value? If so, I'd rather use that and null check.

If there isn't, what you suggest in your comment is fine.

Reply:
```suggestion
			if (brewId == null || !BotaniaAPI.instance().getBrewRegistry().getOrEmpty(brewId).isPresent()) {
```